pkg/plugins/hybrid/v1alpha/scaffolds/init.go

```diff
@@ -125,6 +131,9 @@ func (s *initScaffolder) Scaffold() error {
 		&templates.DockerIgnore{},
 	)
 
+	// Add note to include any depedencies in dockerfile which are required to build `main.go`.
+	fmt.Println("Include any dependencies required to build `main.go` in your project to Dockerfile")
```

Comment:
creating placeholder folders (like api/ and controllers/) will be an issue because when the go api is scaffolded after `init` command, the plugin will error stating that api folder already exists. 
So if we create placeholder repositories, this scenario will cause problems:
1. operator-sdk init --plugin=hybrid
2. operator-sdk create api --plugin=go
One option to make this work is to allow multi group by default in the plugin.

Reply:
Hmm, this is tough. So basically if we don't add those directories on init, the dockerfile can't build, if we do add them, the golang create api won't work?

That's odd though, can't you create multiple apis in a go project? Why wouldn't those subsequent create api calls error out in the same way?

Reply:
Testing locally, the go plugin didn't seem to have any trouble creating the api when the empty directories already existed. This was with operator-sdk version `v1.10.0-2-g82e8ebbc`

Reply:
fixed to scaffold placeholder repositories.